example/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/corazawaf/coraza-coreruleset/v4/example
 go 1.18
 
 require (
-	github.com/corazawaf/coraza-coreruleset v0.0.0-20240226094324-415b1017abdc
+	github.com/corazawaf/coraza-coreruleset/v4 v4.0.0-coraza.1
```

Comment:
I thought we agreed on not having the coraza suffix in https://github.com/corazawaf/coraza-coreruleset/pull/11#issuecomment-1957672081 and instead cut our own patch version.

Reply:
Switched to v4.0.0 ✅